### ISSUE TITLE
Multiple DLCs Missing From Website

### ISSUE THREAD

**Issue by pettta** [connorshea/vglist#3776]
Multiple modern DLCs are missing from the website. I'm not sure if this is intended or a bug, but I thought I would bring up an issue here in case it is a bug. Here are some examples: 

CDProjektRed has the witcher DLCs but not Cyberpunk Phantom Liberty: https://vglist.co/companies/344

Fromsoftware has the dark souls DLCs but not Shadow of the Erdtree: 
https://vglist.co/companies/32

**Comment by connorshea** [comment]
Yeah, there isn't really an official system right now for DLC/expansions, it's been something I've toyed with a bit but haven't actually gone through and properly implemented. So that's why there's usually not coverage of it right now.
